core/src/main/java/com/graphhopper/routing/util/BikeCommonFlagEncoder.java

```diff
@@ -248,8 +248,19 @@ public long acceptWay(ReaderWay way) {
         if (!highwaySpeeds.containsKey(highwayValue))
             return 0;
 
+        String sacScale = way.getTag("sac_scale");
+        if (sacScale != null) {
+            if ((way.hasTag("highway", "cycleway"))
+                    && (way.hasTag("sac_scale", "hiking")))
+                return acceptBit;
+            if (!isSacScaleAllowed(sacScale))
+                return 0;
+        }
+
         // use the way if it is tagged for bikes
-        if (way.hasTag("bicycle", intendedValues) || way.hasTag("bicycle", "dismount"))
+        if (way.hasTag("bicycle", intendedValues) || 
+            way.hasTag("bicycle", "dismount") || 
+           (way.hasTag("highway", "cycleway")))
```

Comment:
Would you mind to remove the double brackets `(way....)`? Also for the above statement